### PR TITLE
python3Packages.cmarkgfm: 0.5.2 -> 0.5.3

### PR DESCRIPTION
**Pull request by fabaff**
###### Motivation for this change
Update to latest upstream release 

Change log: https://github.com/theacodes/cmarkgfm/releases/tag/0.5.3

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by risicle** [comment]
All `nixpkgs-review` failures are preexisting for me, linux x86_64 & macos 10.14

**Review by risicle — APPROVED** [review]

File: pkgs/development/python-modules/semver/default.nix [hunk 1]
```diff
@@ -7,21 +7,23 @@
...
 
   src = fetchFromGitHub {
     owner = "python-semver";
     repo = "python-semver";
     rev = version;
-    sha256 = "0yxjmcgk5iwp53l9z1cg0ajrj18i09ircs11ifpdrggzm8n1blf3";
+    sha256 = "sha256-IWTo/P9JRxBQlhtcH3JMJZZrwAA8EALF4dtHajWUc4w=";
   };
 
   preCheck = "rm -rf dist"; # confuses source vs dist imports in pytest
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/semver/default.nix:19]:
```suggestion
  preCheck = "rm -r dist"; # confuses source vs dist imports in pytest
```

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review with [nixpkgs-review-checks extension](https://github.com/SuperSandro2000/nixpkgs-review-checks). It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).
If you have any questions or problems please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 116308` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>

  <summary>11 packages failed to build and already failed to build on hydra master:</summary>
  <ul>
<li>apache-airflow: log was empty</li>
<li>mnemosyne: log was empty</li>
<li>python38Packages.apispec: log was empty</li>
<li>python38Packages.flask-appbuilder: log was empty</li>
<li>python38Packages.prance: log was empty</li>
<li>python38Packages.restview: log was empty</li>
<li>python38Packages.toggl-cli: log was empty</li>
<li>python39Packages.apispec: log was empty</li>
<li>python39Packages.flask-appbuilder: log was empty</li>
<li>python39Packages.prance: log was empty</li>
<li>python39Packages.restview: log was empty</li>
  </ul>
</details>
<details>

  <summary>45 packages failed to build and are new build failures:</summary>
  <ul>
<li>check_systemd: log was empty</li>
<li>evdevremapkeys: log was empty</li>
<li>hwi: log was empty</li>
<li>(python38Packages.hwi): log was empty</li>
<li>keepkey_agent: log was empty</li>
<li>(python38Packages.keepkey_agent): log was empty</li>
<li>ledger_agent: log was empty</li>
<li>(python38Packages.ledger_agent): log was empty</li>
<li>luigi: log was empty</li>
<li>mailman-web: log was empty</li>
<li>(python38Packages.mailman-web): log was empty</li>
<li>python38Packages.ansible-kernel: log was empty</li>
<li>python38Packages.ansible-runner: log was empty</li>
<li>python38Packages.bitbox02: log was empty</li>
<li>python38Packages.cmarkgfm: log was empty</li>
<li>python38Packages.gtts: log was empty</li>
<li>python38Packages.jupyter-repo2docker: log was empty</li>
<li>python38Packages.labelbox: log was empty</li>
<li>python38Packages.libagent: log was empty</li>
<li>python38Packages.nagiosplugin: log was empty</li>
<li>python38Packages.ndjson: log was empty</li>
<li>python38Packages.pkutils: log was empty</li>
<li>python38Packages.plugwise: log was empty</li>
<li>python38Packages.postorius: log was empty</li>
<li>python38Packages.pygogo: log was empty</li>
<li>python38Packages.python-daemon: log was empty</li>
<li>python38Packages.readme: log was empty</li>
<li>python38Packages.readme_renderer: log was empty</li>
<li>python38Packages.salmon-mail: log was empty</li>
<li>python38Packages.semver: log was empty</li>
<li>trezor_agent: log was empty</li>
<li>(python38Packages.trezor_agent): log was empty</li>
<li>python39Packages.bitbox02: log was empty</li>
<li>python39Packages.cmarkgfm: log was empty</li>
<li>python39Packages.jupyter-repo2docker: log was empty</li>
<li>python39Packages.mailman-web: log was empty</li>
<li>python39Packages.pkutils: log was empty</li>
<li>python39Packages.plugwise: log was empty</li>
<li>python39Packages.postorius: log was empty</li>
<li>python39Packages.pygogo: log was empty</li>
<li>python39Packages.readme: log was empty</li>
<li>python39Packages.readme_renderer: log was empty</li>
<li>python39Packages.semver: log was empty</li>
<li>sublime-music: log was empty</li>
<li>terraform-compliance: log was empty</li>
  </ul>
</details>

The following issues got detected with the above build packages.
Please fix at least the ones listed with your changed packages:

<details>
python38Packages.ansible-kernel:
<details>


warning: python-include-tests
Add a `checkPhase` for tests, or at least `pythonImportsCheck`.

See: https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/python-include-tests.md
</details>
python38Packages.flask-appbuilder:
<details>


warning: python-include-tests
Add a `checkPhase` for tests, or at least `pythonImportsCheck`.

See: https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/python-include-tests.md
</details>
python38Packages.gtts:
<details>


warning: python-explicit-check-phase
Consider using `pytestCheckHook` in `checkInputs` rather than `checkPhase = "pytest";`.

Near pkgs/development/python-modules/gtts/default.nix:41:3:

```
   |
41 |   checkPhase = ''
   |   ^
```

See: https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/python-explicit-check-phase.md
</details>
python38Packages.jupyter-repo2docker:
<details>


warning: unused-argument
Unused argument: `pyyaml`.
Near pkgs/development/python-modules/jupyter-repo2docker/default.nix:7:3:

```
  |
7 | , pyyaml
  |   ^
```
</details>
python38Packages.libagent:
<details>


warning: unclear-gpl
`gpl3` is a deprecated license, check if project uses `gpl3Plus` or `gpl3Only` and change `meta.license` accordingly.

Near pkgs/development/python-modules/libagent/default.nix:28:5:

```
   |
28 |     license = licenses.gpl3;
   |     ^
```

See: https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/unclear-gpl.md
</details>
python38Packages.ndjson:
<details>


warning: license-missing
Package is missing a license

Near pkgs/development/python-modules/ndjson/default.nix:15:3:

```
   |
15 |   meta = with lib; {
   |   ^
```

See: https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/license-missing.md
warning: python-include-tests
Add a `checkPhase` for tests, or at least `pythonImportsCheck`.

See: https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/python-include-tests.md
</details>
python38Packages.pkutils:
<details>


warning: unused-argument
Unused argument: `pythonOlder`.
Near pkgs/development/python-modules/pkutils/default.nix:2:3:

```
  |
2 | , pythonOlder
  |   ^
```
</details>
python38Packages.postorius:
<details>


warning: python-include-tests
Add a `checkPhase` for tests, or at least `pythonImportsCheck`.

See: https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/python-include-tests.md
warning: unused-argument
Unused argument: `mailmanclient`.
Near pkgs/servers/mail/mailman/postorius.nix:2:20:

```
  |
2 | , django-mailman3, mailmanclient, readme_renderer
  |                    ^
```
</details>
python38Packages.prance:
<details>


warning: python-include-tests
Add a `checkPhase` for tests, or at least `pythonImportsCheck`.

See: https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/python-include-tests.md
warning: unused-argument
Unused argument: `sphinx`.
Near pkgs/development/python-modules/prance/default.nix:12:3:

```
   |
12 | , sphinx
   |   ^
```
</details>
python38Packages.pygogo:
<details>


warning: unused-argument
Unused argument: `pytestCheckHook`.
Near pkgs/development/python-modules/pygogo/default.nix:7:3:

```
  |
7 | , pytestCheckHook
  |   ^
```
</details>
python38Packages.python-daemon:
<details>


warning: maintainers-missing
Package does not have a maintainer. Consider adding yourself?

Near pkgs/development/python-modules/python-daemon/default.nix:35:3:

```
   |
35 |   meta = with lib; {
   |   ^
```

See: https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/maintainers-missing.md
</details>
python38Packages.readme:
<details>


warning: python-explicit-check-phase
Consider using `pytestCheckHook` in `checkInputs` rather than `checkPhase = "pytest";`.

Near pkgs/development/python-modules/readme/default.nix:25:3:

```
   |
25 |   checkPhase = ''
   |   ^
```

See: https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/python-explicit-check-phase.md
warning: python-include-tests
Add a `checkPhase` for tests, or at least `pythonImportsCheck`.

See: https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/python-include-tests.md
</details>
python38Packages.restview:
<details>


warning: unclear-gpl
`gpl2` is a deprecated license, check if project uses `gpl2Plus` or `gpl2Only` and change `meta.license` accordingly.

Near pkgs/development/python-modules/restview/default.nix:41:5:

```
   |
41 |     license = lib.licenses.gpl2;
   |     ^
```

See: https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/unclear-gpl.md
warning: python-include-tests
Add a `checkPhase` for tests, or at least `pythonImportsCheck`.

See: https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/python-include-tests.md
</details>
python38Packages.salmon-mail:
<details>


warning: unclear-gpl
`gpl3` is a deprecated license, check if project uses `gpl3Plus` or `gpl3Only` and change `meta.license` accordingly.

Near pkgs/development/python-modules/salmon-mail/default.nix:31:5:

```
   |
31 |     license = licenses.gpl3;
   |     ^
```

See: https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/unclear-gpl.md
</details>
python38Packages.toggl-cli:
<details>


warning: duplicate-check-inputs
Dependencies listed in `propagatedBuildInputs` should not be repeated in `checkInputs`.

Detected duplicates:

- python3.8-pbr-5.5.1

Near pkgs/development/python-modules/toggl-cli/default.nix:38:3:

```
   |
38 |   checkInputs = [ pbr pytest pytestcov pytest-mock faker factory_boy ];
   |   ^
```

Near pkgs/development/python-modules/toggl-cli/default.nix:52:3:

```
   |
52 |   propagatedBuildInputs = [
   |   ^
```

See: https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/duplicate-check-inputs.md
warning: unused-argument
Unused argument: `pythonAtLeast`.
Near pkgs/development/python-modules/toggl-cli/default.nix:1:39:

```
  |
1 | { lib, buildPythonPackage, fetchPypi, pythonAtLeast, pythonOlder
  |                                       ^
```
</details>
python39Packages.flask-appbuilder:
<details>


warning: python-include-tests
Add a `checkPhase` for tests, or at least `pythonImportsCheck`.

See: https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/python-include-tests.md
</details>
python39Packages.jupyter-repo2docker:
<details>


warning: unused-argument
Unused argument: `pyyaml`.
Near pkgs/development/python-modules/jupyter-repo2docker/default.nix:7:3:

```
  |
7 | , pyyaml
  |   ^
```
</details>
python39Packages.mailman-web:
<details>


warning: unclear-gpl
`gpl3` is a deprecated license, check if project uses `gpl3Plus` or `gpl3Only` and change `meta.license` accordingly.

Near pkgs/servers/mail/mailman/web.nix:41:5:

```
   |
41 |     license = licenses.gpl3;
   |     ^
```

See: https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/unclear-gpl.md
warning: python-include-tests
Add a `checkPhase` for tests, or at least `pythonImportsCheck`.

See: https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/python-include-tests.md
</details>
python39Packages.pkutils:
<details>


warning: unused-argument
Unused argument: `pythonOlder`.
Near pkgs/development/python-modules/pkutils/default.nix:2:3:

```
  |
2 | , pythonOlder
  |   ^
```
</details>
python39Packages.postorius:
<details>


warning: python-include-tests
Add a `checkPhase` for tests, or at least `pythonImportsCheck`.

See: https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/python-include-tests.md
warning: unused-argument
Unused argument: `mailmanclient`.
Near pkgs/servers/mail/mailman/postorius.nix:2:20:

```
  |
2 | , django-mailman3, mailmanclient, readme_renderer
  |                    ^
```
</details>
python39Packages.prance:
<details>


warning: python-include-tests
Add a `checkPhase` for tests, or at least `pythonImportsCheck`.

See: https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/python-include-tests.md
warning: unused-argument
Unused argument: `sphinx`.
Near pkgs/development/python-modules/prance/default.nix:12:3:

```
   |
12 | , sphinx
   |   ^
```
</details>
python39Packages.pygogo:
<details>


warning: unused-argument
Unused argument: `pytestCheckHook`.
Near pkgs/development/python-modules/pygogo/default.nix:7:3:

```
  |
7 | , pytestCheckHook
  |   ^
```
</details>
python39Packages.readme:
<details>


warning: python-explicit-check-phase
Consider using `pytestCheckHook` in `checkInputs` rather than `checkPhase = "pytest";`.

Near pkgs/development/python-modules/readme/default.nix:25:3:

```
   |
25 |   checkPhase = ''
   |   ^
```

See: https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/python-explicit-check-phase.md
warning: python-include-tests
Add a `checkPhase` for tests, or at least `pythonImportsCheck`.

See: https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/python-include-tests.md
</details>
python39Packages.restview:
<details>


warning: unclear-gpl
`gpl2` is a deprecated license, check if project uses `gpl2Plus` or `gpl2Only` and change `meta.license` accordingly.

Near pkgs/development/python-modules/restview/default.nix:41:5:

```
   |
41 |     license = lib.licenses.gpl2;
   |     ^
```

See: https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/unclear-gpl.md
warning: python-include-tests
Add a `checkPhase` for tests, or at least `pythonImportsCheck`.

See: https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/python-include-tests.md
</details>
terraform-compliance:
<details>


warning: unused-argument
Unused argument: `python3Packages`.
Near pkgs/applications/networking/cluster/terraform-compliance/default.nix:13:3:

```
   |
13 | , python3Packages
   |   ^
```
</details>
</details>

File: pkgs/development/python-modules/semver/default.nix (hunk 1 above)
Comment (fabaff) [reply pkgs/development/python-modules/semver/default.nix]: Thanks, removed.

**Comment by fabaff** [comment]
Some of packages which fails requires `chardet-4.0.0` (https://github.com/NixOS/nixpkgs/pull/117258).

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review with [nixpkgs-review-checks extension](https://github.com/SuperSandro2000/nixpkgs-review-checks). It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).
If you have any questions or problems please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 116308` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>

  <summary>12 packages failed to build and already failed to build on hydra master:</summary>
  <ul>
<li>apache-airflow: log was empty</li>
<li>mnemosyne: log was empty</li>
<li>python38Packages.apispec: log was empty</li>
<li>python38Packages.flask-appbuilder: log was empty</li>
<li>python38Packages.prance: log was empty</li>
<li>python38Packages.restview: log was empty</li>
<li>python38Packages.toggl-cli: log was empty</li>
<li>python39Packages.apispec: log was empty</li>
<li>python39Packages.flask-appbuilder: log was empty</li>
<li>python39Packages.prance: log was empty</li>
<li>python39Packages.restview: log was empty</li>
<li>python39Packages.toggl-cli: log was empty</li>
  </ul>
</details>
<details>
<summary>
54 packages built:
</summary>
<ul>
<li>azure-cli</li>
<li>check_systemd</li>
<li>evdevremapkeys</li>
<li>hwi (python38Packages.hwi)</li>
<li>keepkey_agent (python38Packages.keepkey_agent)</li>
<li>ledger_agent (python38Packages.ledger_agent)</li>
<li>luigi</li>
<li>mailman-web (python38Packages.mailman-web)</li>
<li>python38Packages.ansible-kernel</li>
<li>python38Packages.ansible-runner</li>
<li>python38Packages.bitbox02</li>
<li>python38Packages.cmarkgfm</li>
<li>python38Packages.gtts</li>
<li>python38Packages.jupyter-repo2docker</li>
<li>python38Packages.labelbox</li>
<li>python38Packages.libagent</li>
<li>python38Packages.nagiosplugin</li>
<li>python38Packages.ndjson</li>
<li>python38Packages.pkutils</li>
<li>python38Packages.plugwise</li>
<li>python38Packages.postorius</li>
<li>python38Packages.pygogo</li>
<li>python38Packages.python-daemon</li>
<li>python38Packages.readme</li>
<li>python38Packages.readme_renderer</li>
<li>python38Packages.salmon-mail</li>
<li>python38Packages.semver</li>
<li>trezor_agent (python38Packages.trezor_agent)</li>
<li>python39Packages.ansible-kernel</li>
<li>python39Packages.ansible-runner</li>
<li>python39Packages.bitbox02</li>
<li>python39Packages.cmarkgfm</li>
<li>python39Packages.gtts</li>
<li>python39Packages.hwi</li>
<li>python39Packages.jupyter-repo2docker</li>
<li>python39Packages.keepkey_agent</li>
<li>python39Packages.labelbox</li>
<li>python39Packages.ledger_agent</li>
<li>python39Packages.libagent</li>
<li>python39Packages.mailman-web</li>
<li>python39Packages.nagiosplugin</li>
<li>python39Packages.ndjson</li>
<li>python39Packages.pkutils</li>
<li>python39Packages.plugwise</li>
<li>python39Packages.postorius</li>
<li>python39Packages.pygogo</li>
<li>python39Packages.python-daemon</li>
<li>python39Packages.readme</li>
<li>python39Packages.readme_renderer</li>
<li>python39Packages.salmon-mail</li>
<li>python39Packages.semver</li>
<li>python39Packages.trezor_agent</li>
<li>sublime-music</li>
<li>terraform-compliance</li>
</ul>
</details>

The following issues got detected with the above build packages.
Please fix at least the ones listed with your changed packages:

<details>
azure-cli:
<details>


warning: stale-substitute
Stale substituteInPlace detected.
substituteStream(): WARNING: pattern 'cryptography>=2.3.1,<3.0.0' doesn't match anything in file 'setup.py'
See: https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/stale-substitute.md
</details>
</details>

**Review by freezeboy — COMMENTED** [review]
I also found some failing packages simply requiring `chardet` and others look good now, I guess to be sure we could rebase this PR on the `chardet` PR and relaunch the build bot

**Comment by fabaff** [comment]
Some failures are fixed:

- prance: https://github.com/NixOS/nixpkgs/pull/117481
- restview : https://github.com/NixOS/nixpkgs/pull/117374

**Comment by fabaff** [comment]
`apache-airflow` is already broken.

```text
1 package failed to build:
apache-airflow

66 packages built:
azure-cli check_systemd evdevremapkeys hwi keepkey_agent ledger_agent luigi mailman-web mnemosyne python38Packages.ansible-kernel python38Packages.ansible-runner python38Packages.apispec python38Packages.bitbox02 python38Packages.cmarkgfm python38Packages.flask-appbuilder python38Packages.gtts python38Packages.jupyter-repo2docker python38Packages.labelbox python38Packages.libagent python38Packages.nagiosplugin python38Packages.ndjson python38Packages.pkutils python38Packages.plugwise python38Packages.postorius python38Packages.prance python38Packages.pygogo python38Packages.python-daemon python38Packages.readme python38Packages.readme_renderer python38Packages.restview python38Packages.salmon-mail python38Packages.semver python38Packages.toggl-cli trezor_agent python38Packages.twine python39Packages.ansible-kernel python39Packages.ansible-runner python39Packages.apispec python39Packages.bitbox02 python39Packages.cmarkgfm python39Packages.flask-appbuilder python39Packages.gtts python39Packages.hwi python39Packages.jupyter-repo2docker python39Packages.keepkey_agent python39Packages.labelbox python39Packages.ledger_agent python39Packages.libagent python39Packages.mailman-web python39Packages.nagiosplugin python39Packages.ndjson python39Packages.pkutils python39Packages.plugwise python39Packages.postorius python39Packages.prance python39Packages.pygogo python39Packages.python-daemon python39Packages.readme python39Packages.readme_renderer python39Packages.restview python39Packages.salmon-mail python39Packages.semver python39Packages.trezor_agent python39Packages.twine sublime-music terraform-compliance
```

File: pkgs/development/python-modules/semver/default.nix [hunk 2]
```diff
@@ -1,27 +1,34 @@
...
 
-  preCheck = "rm -rf dist"; # confuses source vs dist imports in pytest
-  checkInputs = [ pytestCheckHook pytestcov ];
+  checkInputs = [
+    pytest-cov
+    pytestCheckHook
+  ];
+
+  # Confuses source vs dist imports in pytest
+  preCheck = "rm -r dist";
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/semver/default.nix:25]:
```suggestion
  preCheck = ''
    rm -r dist
  '';
```

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review with [nixpkgs-review-checks extension](https://github.com/SuperSandro2000/nixpkgs-review-checks). It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).
If you have any questions or problems please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 116308` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>

  <summary>2 packages failed to build and already failed to build on hydra master:</summary>
  <ul>
<li>apache-airflow: <a href="https://termbin.com/67qe">plain log</a>  |  <a href="https://htmlpreview.github.io/?https://gist.githubusercontent.com/SuperSandroBot/fb62ac1bbda6c1d7815ff201abcfdb72/raw/index.html">pretty log </a></li>
<li>python39Packages.plugwise: log was empty</li>
  </ul>
</details>
<details>
<summary>
63 packages built:
</summary>
<ul>
<li>azure-cli</li>
<li>check_systemd</li>
<li>evdevremapkeys</li>
<li>hwi (python38Packages.hwi)</li>
<li>keepkey_agent (python38Packages.keepkey_agent)</li>
<li>ledger_agent (python38Packages.ledger_agent)</li>
<li>luigi</li>
<li>mailman-web (python38Packages.mailman-web)</li>
<li>mnemosyne</li>
<li>python38Packages.ansible-kernel</li>
<li>python38Packages.ansible-runner</li>
<li>python38Packages.apispec</li>
<li>python38Packages.bitbox02</li>
<li>python38Packages.cmarkgfm</li>
<li>python38Packages.flask-appbuilder</li>
<li>python38Packages.gtts</li>
<li>python38Packages.jupyter-repo2docker</li>
<li>python38Packages.labelbox</li>
<li>python38Packages.libagent</li>
<li>python38Packages.nagiosplugin</li>
<li>python38Packages.ndjson</li>
<li>python38Packages.pkutils</li>
<li>python38Packages.plugwise</li>
<li>python38Packages.postorius</li>
<li>python38Packages.prance</li>
<li>python38Packages.pygogo</li>
<li>python38Packages.python-daemon</li>
<li>python38Packages.readme</li>
<li>python38Packages.readme_renderer</li>
<li>python38Packages.restview</li>
<li>python38Packages.salmon-mail</li>
<li>python38Packages.semver</li>
<li>python38Packages.toggl-cli</li>
<li>trezor_agent (python38Packages.trezor_agent)</li>
<li>python39Packages.ansible-kernel</li>
<li>python39Packages.ansible-runner</li>
<li>python39Packages.apispec</li>
<li>python39Packages.bitbox02</li>
<li>python39Packages.cmarkgfm</li>
<li>python39Packages.flask-appbuilder</li>
<li>python39Packages.gtts</li>
<li>python39Packages.hwi</li>
<li>python39Packages.jupyter-repo2docker</li>
<li>python39Packages.keepkey_agent</li>
<li>python39Packages.labelbox</li>
<li>python39Packages.ledger_agent</li>
<li>python39Packages.libagent</li>
<li>python39Packages.mailman-web</li>
<li>python39Packages.nagiosplugin</li>
<li>python39Packages.ndjson</li>
<li>python39Packages.pkutils</li>
<li>python39Packages.postorius</li>
<li>python39Packages.prance</li>
<li>python39Packages.pygogo</li>
<li>python39Packages.python-daemon</li>
<li>python39Packages.readme</li>
<li>python39Packages.readme_renderer</li>
<li>python39Packages.restview</li>
<li>python39Packages.salmon-mail</li>
<li>python39Packages.semver</li>
<li>python39Packages.trezor_agent</li>
<li>sublime-music</li>
<li>terraform-compliance</li>
</ul>
</details>

**Pull request merged**